node/get_status_node.go

```diff
@@ -334,7 +313,6 @@ func (n *StatusNode) stop() error {
 	n.appMetricsSrvc = nil
 	n.walletSrvc = nil
 	n.localNotificationsSrvc = nil
-	n.personalSrvc = nil
```

Comment:
Looks like this makes the personal service unused, let's delete it then?
https://github.com/status-im/status-go/blob/f281ca0a805b5482fc071bf9760e26e18b02118d/services/personal/service.go#L13-L16

Though we keep using the `PublicAPI` of it, but we create it independently and use it in 2 places:
https://github.com/status-im/status-go/blob/e478786e7df17f74d80d966542d38d25e57565ef/api/geth_backend.go#L146 https://github.com/status-im/status-go/blob/4f4be1a78a0a8d3bae1ec627209ef3daa28663e9/services/ext/service.go#L400

Then I suppose we need to extract this type from the service? 🤔 
Not sure what's best here.

Reply:
@igor-sirotin but just got a better idea, why don't we keep the service, and move api logic there (as it should be actually) and use the service in the code instead of api instance, and still provide correct endpoints. Going to do that change.

Reply:
@igor-sirotin an idea from my previous comment implemented, please have a look.